editor.planx.uk/src/ui/editor/MoreInformation/MoreInformation.tsx

```diff
@@ -17,7 +17,7 @@ export const MoreInformation = ({
 }: MoreInformationProps) => {
   return (
     <ModalSection>
-      <ModalSectionContent title="More Information" Icon={InfoOutlined}>
+      <ModalSectionContent title="More information" Icon={Help}>
```

Comment:
This came from feedback here - editor modal now aligned to same icon as frontend, and header casing updated as well to our normal style: https://opensystemslab.slack.com/archives/C07F7215W7P/p1733129391086059